src/wp-includes/option.php

```diff
@@ -1659,20 +1575,7 @@ function delete_network_option( $network_id, $option ) {
 	if ( ! is_multisite() ) {
 		$result = delete_option( $option );
 	} else {
-		$row = $wpdb->get_row( $wpdb->prepare( "SELECT meta_id FROM {$wpdb->sitemeta} WHERE meta_key = %s AND site_id = %d", $option, $network_id ) );
-		if ( is_null( $row ) || ! $row->meta_id ) {
-			return false;
-		}
-		$cache_key = "$network_id:$option";
-		wp_cache_delete( $cache_key, 'site-options' );
-
-		$result = $wpdb->delete(
-			$wpdb->sitemeta,
-			array(
-				'meta_key' => $option,
-				'site_id'  => $network_id,
-			)
-		);
+		$result = delete_metadata( 'site', $network_id, $option, '' );
```

Comment:
```suggestion
		$result = delete_metadata( 'site', $network_id, wp_slash( $option ) );
```

Default not needed (nit pick)

ref:
https://github.com/WordPress/wordpress-develop/blob/deafd1193c244acfae80bb4507de4d8b532fa9e5/src/wp-includes/meta.php#L398

Reply:
👍 Fixed in https://github.com/WordPress/wordpress-develop/pull/2478/commits/faf732ada8e0f3092ffabb3e7d013b778dfade39